pkg/builder/controller.go

```diff
@@ -148,6 +148,12 @@ func (blder *Builder) WithOptions(options controller.Options) *Builder {
 	return blder
 }
 
+// WithLogger overrides the controller options's logger used.
+func (blder *Builder) WithLogger(log logr.Logger) *Builder {
+	blder.log = log
+	return blder
+}
+
```

Comment:
Any particular reason to move this method, or just an artifact of the rebase?

Reply:
Both 😂, moved it up at the bottom of the other `With..` methods